dashboard/src/modules/components/OverviewComponent/common-component.jsx

```diff
@@ -33,7 +40,7 @@ export const NoExpiringRuns = () => {
       <TextContent className="no-runs-wrapper">
         <Text component={TextVariants.h3}> You have no runs expiring soon</Text>
         <Text component={TextVariants.p}>
-          Runs that have expiration date within next 10days will appear here.
+          Runs that have expiration date within next 20 days will appear here.
```

Comment:
It'd be nice to use `{EXPIRATION_DAYS_LIMIT}` here rather than an inline number; if it changes, we don't need to remember to change it in both places.

(Also, at some point, we probably want to make this a user configuration setting rather than a hardcoded value; this change will at least make the dependency visible to code.)

Reply:
Updated!